com.unity.sg2/Editor/GraphUI/ShaderGraphStencil.cs

```diff
@@ -134,43 +133,6 @@ bool IsExposable(TypeHandle typeHandle)
                     action = () =>
                     {
                         var command = new CreateGraphVariableDeclarationCommand(displayName, true, type, typeof(GraphDataVariableDeclarationModel), selectedGroup ?? GraphModel.GetSectionModel(sectionName));
-                        command.InitializationCallback = (decl, constant) =>
```

Comment:
Now that we're not using the initialization callback, I saw this was moved to `InstantiateVariableDeclaration`, (which I approve of btw, not a huge fan of us doing stuff like this outisde `ShaderGraphModel`), can we remove our GTF edit to add the initialization callback to the command?

Reply:
Unless we want to keep it anyway as a suggestion for what GTF do to make its API more symmetrical, cause it is weird that wasn't exposed in the first place, given that the searcher exposes a callback for node initialization